README.md

```diff
@@ -46,20 +46,14 @@ cargo run -p carol -- --cfg carol.yml run
 First compile it to wasm. Here we just compile one of the examples in `example-crates`.
 
 ``` sh
-./build_example.sh bitmex_guest
+wasm_output=$( cargo run -p carlo -- build -p bitmex_guest )
```

Comment:
maybe for the example we should `cargo install --path carlo` and cd into the `bitmex_guest` or whatever directories and run carlo naturally.

Reply:
you mean `nix shell` ;-)

i was thinking about something like that, but upon reflection concluded the obscurity of the cargo run variant vs. the chances that users will run into an issue trying to install  meant robust was more conservative